src/megaapi_impl.cpp

```diff
@@ -13367,27 +13370,55 @@ void MegaApiImpl::syncupdate_local_lockretry(bool waiting)
 }
 #endif
 
-void MegaApiImpl::backupput_result(const Error&, handle backupId)
+void MegaApiImpl::backupput_result(const Error& e, handle backupId)
 {
+    if (e || backupId == UNDEF)
+    {
+        LOG_err << "backupput_result failed: " << MegaError::getErrorString(e);
+    }
+
```

Comment:
It's actually not needed, all failed (and successful) requests result in a log message with the result: success or error code :)